app/scripts/app.js

```
@@ -337,6 +336,56 @@
+        }
+      }
+    } catch (e) {
+      /* empty */
+    }
+  }
+  return 'warn';
+}
+
+async function getUserType(rolesFactory) {
+  try {
+    let response = await fetch('/auth/who_am_i');
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'response' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)